tensorflow/tensorflow.bzl

```diff
@@ -316,29 +316,29 @@ def _cuda_copts():
     common_cuda_opts = ["-x", "cuda", "-DGOOGLE_CUDA=1"]
     return select({
         "//conditions:default": [],
-        "//third_party/gpus/cuda:using_nvcc": (
+        "@local_config_cuda//cuda:using_nvcc": (
             common_cuda_opts +
             [
                 "-nvcc_options=relaxed-constexpr",
```

Comment:
I have opened #3980 for improvements to the CUDA autoconf. If there are any feature requests for `cuda_configure`, feel free to add them there.